env.d/development/localtunnel.dist

```diff
@@ -0,0 +1,5 @@
+## Locatunnel configuration
+
+LOCALTUNNEL_SUBDOMAIN=dev-${USER}-joanie
+LOCALTUNNEL_PORT=8071
```

Comment:
Maybe a nitpick (and I didn't think of it in marsha), but this variable could be called with something like `APP_PORT` or `APP_EXTERNAL_PORT` ?
WDYT ?

Reply:
Seems legit :slightly_smiling_face: